package.json

```diff
@@ -4,6 +4,17 @@
   "description": "JSON-Schema + Faker",
   "homepage": "http://json-schema-faker.js.org",
   "main": "lib/index.js",
+  "scripts": {
```

Comment:
well, I never used npm-scripts in favor of grunt, but let's give it a try.
I must say I still don't clearly understand what `grunt parts` does so I hope this one would be cleaner for me. I will let you know if I see any problems with npm scripts.